modules/src/core/ics04_channel/handler/send_packet.rs

```diff
@@ -32,7 +32,8 @@ pub fn send_packet(ctx: &dyn ChannelReader, packet: Packet) -> HandlerResult<Pac
         return Err(Error::channel_closed(packet.source_channel));
     }
 
-    let _channel_cap = ctx.authenticated_capability(&packet.source_port)?;
+    // Fixme(hu55a1n1)
```

Comment:
To fix this, we would need `Ics20Context` to have access to a scoped `CapabilityReader` and since there's already a PR for ICS20 impl, I updated the PR author checklist to include this -> https://github.com/informalsystems/ibc-rs/pull/1989.